etcdctlv3/command/watch_command.go

```diff
@@ -98,9 +98,15 @@ func recvLoop(wStream pb.Watch_WatchClient) {
 		if err != nil {
 			ExitWithError(ExitError, err)
 		}
-		evs := resp.Events
-		for _, ev := range evs {
-			fmt.Printf("%s: %s %s\n", ev.Type, string(ev.Kv.Key), string(ev.Kv.Value))
+
+		switch {
```

Comment:
I will add a todo. We need to handle more resp type in the future like Cancel and ProgressNotification(even if there is no events sent to a watcher, we still send progress notification to that watcher so that client know where to start after a failure)